middlewares/validators/user.js

```diff
@@ -192,10 +193,31 @@ async function validateUserQueryParams(req, res, next) {
   }
 }
 
+/**
+ * Validator function for query params for the filter route
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ * @param next {Object} - Express middleware function
+ */
+const validateImageVerificationQuery = async (req, res, next) => {
+  const { type: imageType } = req.query;
+  try {
+    if (!IMAGE_VERIFICATION_TYPES.includes(imageType)) {
+      throw new Error("Invalid verification type was provided!");
+    }
```

Comment:
what if user passes extra keys which were not needed. and this doesn't validates the type of value acceptable for `type `. someone may pass it a boolean value instead of string.

Reply:
Didnt get you??
This will be strictly mentioned in the API contract